### PR TITLE
feat: upgrade sdk

### PR DESCRIPTION
**Pull request by ledniy**

### CODE REVIEW

File: metrics/gc.js [hunk 1]
```diff
@@ -2,14 +2,17 @@
...
+ */
 module.exports = (meter, {prefix, labels, gcDurationBuckets}) => {
-  const boundaries = gcDurationBuckets || DEFAULT_GC_DURATION_BUCKETS
+  // const boundaries = gcDurationBuckets || DEFAULT_GC_DURATION_BUCKETS
 
   const histogram = meter.createHistogram(prefix + NODEJS_GC_DURATION_SECONDS, {
-    description: 'Garbage collection duration by kind, one of major, minor, incremental or weakcb.',
-    boundaries
+    description: 'Garbage collection duration by kind, one of major, minor, incremental or weakcb.'
   })
```
Comment (ledniy) [inline metrics/gc.js]: @marcbachmann, I believe there's no way to pass it anymore

Comment (marcbachmann) [reply metrics/gc.js]: The annotation is still outdated: https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/opentelemetry-api-metrics/src/types/Meter.ts#L85

But it returns an ObservableGauge instance, which has register functions (Observable interface):
https://github.com/open-telemetry/opentelemetry-js/commit/ba3e3207edd217366710e10a769ba7ad4c4fa655#diff-c9a82ab13ebd74152ffa3cc91afa636f11f19174034c8bc12990ae2a69c2c507R11-R14

https://open-telemetry.github.io/opentelemetry-js/interfaces/_opentelemetry_api_metrics.Observable.html

edit: sorry, this was about the gauge interface

Comment (marcbachmann) [reply metrics/gc.js]: regarding boundaries, this is getting complicated 🙃 
https://github.com/open-telemetry/opentelemetry-js/issues/2999

It now belongs into a view configuration, so maybe we can just leave it out like you've done: https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/opentelemetry-sdk-metrics-base/test/MeterProvider.test.ts#L181-L184

**Comment by marcbachmann** [comment]
we've upgraded that in #9

**Pull request closed without merging**